lib/puppet/reference/configuration.rb

```diff
@@ -22,10 +22,16 @@
 
     # Now print the data about the item.
     val = object.default
-    if name.to_s == "vardir"
-      val = "/opt/puppetlabs/puppet/cache"
-    elsif name.to_s == "confdir"
-      val = "/etc/puppetlabs/puppet"
+    if name.to_s == 'vardir'
+      val = 'Unix/Linux: /opt/puppetlabs/puppet/cache -- Windows: C:\ProgramData\PuppetLabs\puppet\cache -- Non-root user: ~/.puppetlabs/opt/puppet/cache'
```

Comment:
So... changing it to %ALLUSERSPROFILE% wouldn't fix the concern about win 2003 in the first place, right? And we're still fine with leaving it behind in this particular place, right? 

Should we change it to %ALLUSERSPROFILE% regardless? I consider that more opaque, but I don't have any idea how many windows installs have that directory in a place other than C:\ProgramData; if it's a bunch, then sure?